File: cppVersion/scripts-SRC/Ohms_Law.cpp
#include <iostream>
using namespace std;

int voltageOhm(){
    int R, I;
    cout << "What is the resistance value?:" << endl;
    cin >> R;
    cout << "And the current value?" << endl;
    cin >> I;

    return R*I;
}
int resistanceOhm(){
    int V, I;
    cout << "What is the voltage value?:" << endl;
    cin >> V;
    cout << "And the current value?" << endl;
    cin >> I;

    return V/I;
}
int currentOhm(){
    int V, R;
    cout << "What is the voltage value?:" << endl;
    cin >> V;
    cout << "And the resistance value?" << endl;
    cin >> R;

    return V/R;
}

int main(){
    cout << "Do you want to calculate the voltage(1), the resistance(2), or the current(3)?: " << endl;

    int option;
    cin >> option;
    
    switch (option)
    {
    case 1:
        cout << "Your choice: Voltage\n" << endl;
        cout << voltageOhm() << " V" << endl;
        break;
    case 2:
        cout << "Your choice: Resistance\n" << endl;
        cout << resistanceOhm() << " R" << endl;
        break;
    case 3:
        cout << "Your choice: Current\n" << endl;
        cout << currentOhm() << " A" << endl;
        break;
    
    default:
        cout << "I\'m sorry, that option isn\'t ready yet" << endl;
        break;
    }

    return 0;
}

File: electricalCalculator/resources/rawScripts/Ohms_Law.py
# Aquí ejecutamos la calculadora de ley de Ohm
def ley_ohm():
    try:
        op = int(input('Do you want to calculate the voltage(1), the resistance(2), or the current(3)?: '))
    except:
        print('Please, I haven\'t understood you')
        return ley_ohm()
    if op == 1:
        print('Your choice: Voltage\n')
        return voltage_ohm()
    elif op == 2:
        print('Your choice: Resistance\n')
        return resistance_ohm()
    elif op == 3:
        print('Your choice: Current\n')
        return current_ohm()
    else:
        print('I\'m sorry, that option isn\'t ready yet')
        return ley_ohm()

def voltage_ohm():
    R, I = float(input('What is the resistance value?:\n')), float(input('And the current value?\n'))
    V = R * I
    print('The voltage are', str(V) + 'V')
    return

def resistance_ohm():
    V, I = float(input('What is the voltage value?:\n')), float(input('And the current value?\n'))
    R = V / I
    print('The resistance are', str(R) + 'Ω')
    return

def current_ohm():
    V, R = float(input('What is the voltage value?:\n')), float(input('And the resistance value?\n'))
    I = V / R
    print('The current are', str(I) + 'A')
    return

ley_ohm()